### PR TITLE
Entroducing CQC kit. A syndicate tactical melee combat dedicaded drip 

### PR DESCRIPTION
**Pull request by ElPidor**
The holy syndicate soldier 
image
image
image
image
image

Why balance 
Its funny. Mostly syndicates will use stunmeta or slipping meta, but not chad close combat. So this set of armor will help them 
## Breaking changes
I made DisarmBonus
You add bonus to thing and if you shove you get bonus to shove mainly gloves
but this shit for some reason doesnt work on gloves but if you add this to Urist and shove itll work 
the more it goes to 0 the more % of succes will be 
I also made disarming event so game should understand if disarmer has bonus or not  and then apply it in the end but....
I made new glasses (god help) You can toggle them but i dont know how to make that when you toggle lights sick .ogg will play but not standart flashlight.ogg
I made new suit
I made new belt 
I made new gloves (god help me)
I made new helmet 
I made new silenced boots
I made 3 new sound for this set
**Changelog**
I hope itll be good

TO DO 

- [ ] Cool sound after toggling lights 
- [ ] Change suit layers so goggles sprite will go above any other spirte (This might incourage player not to change their armor)
- [ ] Make a full working BonusDisarming thing  The heart of set shove\disarming gambling, so players will try to disarm their enemies or running from them
- [ ] Put it uplink
- [ ] Polish i guess?

## Release Notes
The Gorlex LLC and Waffle.CO diceded to make a sick syndicate drip that is bad against bullets but good againts blunt and etc
- **New Features**
  - Introduced `DisarmBonusComponent` to enhance disarming mechanics.
  - Added new entities including:
    - `ClothingBeltSyndieWest` (compact chest rig).
    - `ClothingEyesSyndieVisor` (visor with unique effects).
    - `ClothingHandsSyndieCombat` (close quarters potholders).
    - `ClothingSyndieLeatherMask` (armored mask).
    - `ClothingUniformJumpsuitSyndieass` (grey jumpsuit).
    - `ClothingShoesSyndieBoots` (silenced tactical boots).
  - New sound collection for silenced footsteps.

- **Bug Fixes**
  - Adjusted disarm chance calculations for improved gameplay.

- **Documentation**
  - Added metadata files for new texture resources.

### CODE REVIEW

**Comment by ElPidor** [comment]
WHAIT WHAIT WHAIT 
NO NO NO NO NO
WRONG repo???

**Comment by ElPidor** [comment]
Okey, i did allright time to clean all stuff above

File: Resources/Audio/Effects/Footsteps/silencedbootsstep.ogg
Comment (Piras314) [inline Resources/Audio/Effects/Footsteps/silencedbootsstep.ogg:1]: Should be in _Goobstation folder

File: Resources/Audio/Items/visor_off.ogg
Comment (Piras314) [inline Resources/Audio/Items/visor_off.ogg:1]: Should be in _Goobstation folder

File: Resources/Audio/Items/visor_on.ogg
Comment (Piras314) [inline Resources/Audio/Items/visor_on.ogg:1]: Should be in _Goobstation folder, also credit isn't added for any of these audios it doesn't look like

File: Resources/Prototypes/Entities/Clothing/Belt/belts.yml
Comment (Piras314) [inline Resources/Prototypes/Entities/Clothing/Belt/belts.yml:1]: All the changes here should be in the corresponding file in a _Goobstation folder (create one if there isn't one already)

File: Resources/Prototypes/Entities/Clothing/Belt/belts.yml [hunk 1]
```diff
@@ -712,3 +712,18 @@
     whitelist:
       tags:
       - WizardWand
+
+- type: entity
+  parent: [ClothingBeltStorageBase, ContentsExplosionResistanceBase, BaseSyndicateContraband]
+  id: ClothingBeltSyndieWest
+  name: compact chest rig
+  description: A brand new chest rig for an operation. Gorlex LLC provides it as an addition to compact armorsuit.
```
Comment (Piras314) [inline Resources/Prototypes/Entities/Clothing/Belt/belts.yml:720]: change to "A branch new chest rig provided by Gorlex LLC in addition to the compact armor suit"

File: Resources/Prototypes/Entities/Clothing/Belt/belts.yml [hunk 2]
```diff
@@ -712,3 +712,18 @@
     whitelist:
       tags:
       - WizardWand
+
+- type: entity
+  parent: [ClothingBeltStorageBase, ContentsExplosionResistanceBase, BaseSyndicateContraband]
+  id: ClothingBeltSyndieWest
```
Comment (Piras314) [inline Resources/Prototypes/Entities/Clothing/Belt/belts.yml:718]: Vest, not West

**Review by Piras314 — CHANGES_REQUESTED** [review]
didn't review everything but apply what I said everywhere else and where I said and I'll review it again after

**Comment by Piras314** [comment]
Also check the checkfails.

**Comment by JoeHammad1844** [comment]
1. im not a fan of the sprites, they are all too dark
2. also, make the bundle come in the syndicate backpack so you dont need the extra purchase for full move speed
3. please dont resprite the syndicate recruit jumpsuit, that one is for ship vs ship and i'd prefer it stays that way, make a whole new jumpsuit prototype instead

**Comment by ElPidor** [comment]
> 1. im not a fan of the sprites, they are all too dark
> 2. also, make the bundle come in the syndicate backpack so you dont need the extra purchase for full move speed
> 3. please dont resprite the syndicate recruit jumpsuit, that one is for ship vs ship and i'd prefer it stays that way, make a whole new jumpsuit prototype instead
1 Yeah i agree, i made it a little more colourfull
image
.  you can dm or just write in generall Im bloodyspy on discord. 
2 Yeah i guess, but ill try to check is if it possible to put all bundle into vest, because its all "compact" styled
3 I didnt resprite it, like i did but then i made syndidrip.rsi 
image
Itll come with a bundle or ill ad it as pointless item\emmagged clothesmate item 
image
image

**Comment by InfinityPandaRed** [comment]
> > 1. im not a fan of the sprites, they are all too dark
> > 2. also, make the bundle come in the syndicate backpack so you dont need the extra purchase for full move speed
> > 3. please dont resprite the syndicate recruit jumpsuit, that one is for ship vs ship and i'd prefer it stays that way, make a whole new jumpsuit prototype instead
> >    1 Yeah i agree, i made it a little more colourfull
> >    image
> >    .  you can dm or just write in generall Im bloodyspy on discord.
> >    2 Yeah i guess, but ill try to check is if it possible to put all bundle into vest, because its all "compact" styled
> >    3 I didnt resprite it, like i did but then i made syndidrip.rsi
> >    image
> >    Itll come with a bundle or ill ad it as pointless item\emmagged clothesmate item
> >    image
> >    image

yeah that looks way better

**Comment by ElPidor** [comment]
Since new stamina meta got merged that is pretty good for my PR 
The bad side is that i wont be able physically continue any of my ideas for a year. 
The concet is yours, dear ananymouses, i hope you wont be lazy and will make my dream PR real. BTW, i made odd sounds so dont be afraid about license. Wait me for next year and ill make _syndicate look grate again_
     - With honor, BloodySpy

**Pull request closed without merging**